### PR TITLE
Documented to start Canton via java directly

### PR DESCRIPTION
**Pull request by soren-da**
Instead of always relying on `bin/canton` to start Canton, we now also document to start Canton via `java` in the getting started guide.

### CODE REVIEW

**Comment by marciw-da** [comment]
carrielaben-da is on PTO so added daml-docs for review instead.

**Comment by garyverhaegen-da** [comment]
Could we specifically document the need for the BouncyCastle jar? The current documentation feels misleading, and I don't think it should rely on Rémy's hack. I'd much rather we document how to properly start things with `-cp`.

File: docs/2.7.0/docs/canton/tutorials/getting_started.rst [hunk 1]
```diff
@@ -81,7 +79,11 @@ Then, run
 
    bin/canton --help
 
-to see the command line options that Canton supports.
+to see the command line options that Canton supports. Alternatively to
+`bin/canton` you can also start Canton directly with `java -jar
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/canton/tutorials/getting_started.rst:83]:
```suggestion
``bin/canton``, you can also start Canton directly with ``java -jar
```

File: docs/2.7.0/docs/canton/tutorials/getting_started.rst [hunk 2]
```diff
@@ -81,7 +79,11 @@ Then, run
...
+lib/canton-*.jar` assuming all other jar dependencies are in the `lib` folder
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/canton/tutorials/getting_started.rst:84]:
```suggestion
lib/canton-*.jar``, assuming all other jar dependencies are in the ``lib`` folder,
```

File: docs/2.8.0/docs/canton/tutorials/getting_started.rst [hunk 3]
```diff
@@ -81,7 +79,11 @@ Then, run
 
    bin/canton --help
 
-to see the command line options that Canton supports.
+to see the command line options that Canton supports. Alternatively to
+`bin/canton` you can also start Canton directly with `java -jar
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/canton/tutorials/getting_started.rst:83]:
```suggestion
``bin/canton``, you can also start Canton directly with ``java -jar
```

File: docs/2.8.0/docs/canton/tutorials/getting_started.rst [hunk 4]
```diff
@@ -81,7 +79,11 @@ Then, run
...
+lib/canton-*.jar` assuming all other jar dependencies are in the `lib` folder
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/canton/tutorials/getting_started.rst:84]:
```suggestion
lib/canton-*.jar``, assuming all other jar dependencies are in the ``lib`` folder,
```

**Review by loiskilberg-da — APPROVED** [review]
Looks good! Just added some small formatting suggestions.

**Comment by soren-da** [comment]
> Could we specifically document the need for the BouncyCastle jar? The current documentation feels misleading, and I don't think it should rely on Rémy's hack. I'd much rather we document how to properly start things with `-cp`.

Remy's approach is not a hack, but [standard way](https://docs.oracle.com/javase/tutorial/deployment/jar/downman.html) to specify the JAR's dependency in the manifest, and it works automatically with all fat jar excluded dependencies. Listing the BC jar explicitly and start canton via `java -cp` has a few issues that can break the docs in the future: 1) if the BC jar name changes e.g. due to a version upgrade, 2) if the Canton main class name changes, because we have to specify it when starting via `java -cp`, 3) if we need to add further fat jar excluded jars.

**Review by daravep — APPROVED** [review]
LGTM

**Pull request merged**